app/views/directives/notifications/notification-body.html

```diff
@@ -0,0 +1,68 @@
+<div
+  uib-dropdown
+  class="dropdown pull-right dropdown-kebab-pf"
+  ng-if="notification.actions.length">
+  <button
+    uib-dropdown-toggle
+    class="btn btn-link dropdown-toggle"
+    type="button"
+    id="dropdownKebabRight-{{$id}}"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="true">
+    <span class="fa fa-ellipsis-v"></span>
+  </button>
+  <ul
+    class="dropdown-menu dropdown-menu-right"
+    aria-labelledby="dropdownKebabRight">
+    <li
+      ng-repeat="action in notification.actions"
+      role="{{action.isSeparator === true ? 'separator' : 'menuitem'}}"
+      ng-class="{'divider': action.isSeparator === true, 'disabled': action.isDisabled === true}">
+      <a
+        ng-if="!action.isSeparator"
+        class="secondary-action"
+        title="{{action.title}}"
+        ng-click="$ctrl.customScope.handleAction(notification, action)">
+        {{action.name}}
+      </a>
+    </li>
+  </ul>
+</div>
+
+
+<span
+  ng-if="notification.event"
+  class="{{'pull-left ' + $ctrl.customScope.getNotficationStatusIconClass(notification.event)}}"
+  ng-click="$ctrl.customScope.markRead(notification)"></span>
+<div class="drawer-pf-notification-content">
+  <div
+    class="drawer-pf-notification-message"
+    tooltip-append-to-body="true"
+    tooltip-popup-delay="500"
+    tooltip-placement="bottom"
+    tooltip="{{notification.event.message}}"
+    ng-click="$ctrl.customScope.markRead(notification)">
+    <span>
+      {{notification.event.reason | sentenceCase}} for {{notification.event.involvedObject.kind | humanize}}
+    </span>
+    <a ng-href="{{notification.event | naviateEventInvolvedObjectURL}}">
+      {{notification.event.involvedObject.name}}
+    </a>
+    <span
+      ng-if="notification.event.count > 1"
+      class="text-muted small">
+       <br />{{notification.event.count}} times in the last
+      <duration-until-now timestamp="notification.event.firstTimestamp" omit-single="true" precision="1"></duration-until-now>
+    </span>
+  </div>
+
+  <span ng-if="$ctrl.drawerExpanded" class="drawer-pf-notification-message text-muted small">
+    {{notification.event.message}}
+  </span>
+  <div class="drawer-pf-notification-info"
+    ng-click="$ctrl.customScope.clear(notification)">
+    <span class="date">{{notification.event.lastTimestamp | date:'MM/dd/yyyy'}}</span>
+    <span class="time">{{notification.event.lastTimestamp | date:'h:mm:ss a'}}</span>
```

Comment:
Let's try to use the localized date and time formats if we can (see my comment from last review)